modules/match-media-mock.js

```diff
@@ -0,0 +1,86 @@
+var clone = require('lodash/lang/clone');
+var includes = require('lodash/collection/includes');
+var pull = require('lodash/array/pull');
+var mapValues = require('lodash/object/mapValues');
+var forOwn = require('lodash/object/forOwn');
+var mediaQuery = require('css-mediaquery');
+var isNode = require('detect-node');
+
+var config = {};
+var createdMqls = {};
+
+function matchMediaMock(query) {
```

Comment:
as @alexlande said, this isn't bad, but I'd like to figure out our isomorphic solution before we add this specific of mock code and take a dependency on css-mediaquery